src/Symfony/Component/VarExporter/Tests/VarExporterTest.php

```diff
@@ -248,3 +252,13 @@ public function setFlags($flags)
         throw new \BadMethodCallException('Calling MyArrayObject::setFlags() is forbidden');
     }
 }
+
+class GoodNight
+{
+    public function __sleep()
+    {
+        $this->good = 'night';
```

Comment:
shouldn't this avoid using a dynamic property (unsetting a defined one instead) ? Otherwise, it should set the PHP 8.1 attribute